content/en/building/guides/performance/replication.md

```diff
@@ -140,7 +155,44 @@ A user with `roles: ["role1", "role2", "role3"]` will have a replication depth o
 ``` 
 A user with `roles: ["role1", "role2", "role3"]` will have a replication depth of 5 and report replication depth of 2.
 
+##### Replicate primary contacts
+
+_As of 4.18.0._   
+When a `replication_depth` sets `replicate_primary_contacts` to `true`, the corresponding users will replicate the primary contacts of the places they are allowed to see, even if those contacts are deeper than these users are allowed to download, and even if these contacts belong to other branches of the hierarchy. The primary contact is treated as if it is at the same depth as the place, and will follow the report replication depth of the place. 
+
+For example, considering the following settings:
+```json
+{
+  "replication_depth": [
+    {
+      "role": "chw",
+      "depth": 2,
+      "report_depth": 2,
+      "replicate_primary_contacts": true
+    },
+    {
+      "role": "supervisor",
+      "depth": 2,
+      "report_depth": 1,
+      "replicate_primary_contacts": true
+    }
+  ],
+  "contact_types": [
+    { "id": "level1" },
+    { "id": "level2", "parents": ["level1"] },
+    { "id": "level3", "parents": ["level2"] },
+    { "id": "level4", "parents": ["level3"] },
+    { "id": "level5", "parents": ["level4"] },
+    { "id": "person", "parents": ["level1", "level2", "level3", "level4", "level5"], "person": true s}
+  ]
+}
+```
+
+For he configuration above, a user that has the role `chw` is allowed to replicate contacts 2 levels below their facility and replicate reports for contacts 2 levels below their facility and replicate primary contacts. For a `chw` at `level2`, it will replicate contacts at `level2`, `level3` and `level4`, along with their reports. They will also replicate the primary contact of a place at `level4`, even if the primary contact is a child of a `level5` contact, along with this contact's reports.     
```

Comment:
```suggestion
For the configuration above, a user that has the role `chw` is allowed to replicate contacts 2 levels below their facility and replicate reports for contacts 2 levels below their facility and replicate primary contacts. For a `chw` at `level2`, it will replicate contacts at `level2`, `level3` and `level4`, along with their reports. They will also replicate the primary contact of a place at `level4`, even if the primary contact is a child of a `level5` contact, along with this contact's reports.     
```